.jenkins-scripts/jenkins-common.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+pwd
+
+if [ -z "${GPUDATA+x}" ]; then
+        echo "GPUDATA variable is not set, are we running in Jenkins?"
+        echo "Bailing!"
+        exit 1
+fi
+
+GPU01="$(echo "${GPUDATA}" | cut -d"," -f1 | cut -d"-" -f1)"
+export GPU01
+BUS01="$(echo "${GPUDATA}" | cut -d"," -f1 | cut -d"-" -f2)"
+export BUS01
+GPU02="$(echo "${GPUDATA}" | cut -d"," -f2 | cut -d"-" -f1)"
+export GPU02
+BUS02="$(echo "${GPUDATA}" | cut -d"," -f2 | cut -d"-" -f2)"
+export BUS02
```

Comment:
I found this code block slightly confusing at first glance, and wasn't clear why we had two GPUs specifically.

Can we add two comments here?
1. Explain the format of the GPUDATA variable we're expecting. (Just in case someone wants to build their own test rig, or we have to rebuild ours based on this information.)
1. Explain that we are expecting two GPUs, but the second is only used for the "full" install during nightly builds.